### ISSUE TITLE
nuxt generate --development

### ISSUE THREAD

**Issue by joernroeder** [nuxt/nuxt.js#4117]
### What problem does this feature solve?

I'm having different behavior in my static build compared to the server side rendered version and debugging that without devtools enabled is hard. also validating vuex store e.g. to see if payloads got passed in correctly would be way easier.

### What does the proposed changes look like?

`NODE_ENV` should be regcognized and used if found. `nuxt generated --development` could also build in dev mode

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c7973">#c7973</a>)</em></sub></div>

**Comment by manniL** [comment]
Hey :wave: 

Add the following part to your `nuxt.config.js` to enable debugging:
```js
export default: {
  vue: {
    config: {
      devtools: true
    }
  }
}
```

**Comment by manniL** [comment]
However, the `--development` idea is nice!

I think a PR for this (as the workaround is now known) would be neat :relaxed:

**Comment by manniL** [comment]
resolved by #4357
